.github/workflows/labeler.yml

```
@@ -0,0 +1,11 @@
+name: Autolabeler
```

```suggestion
name: PR Autolabeler
```
For better distinction in the list of all workflows